server/src/internalClusterTest/java/org/opensearch/indices/replication/SearchReplicaReplicationIT.java

```diff
@@ -27,7 +33,7 @@ public class SearchReplicaReplicationIT extends SegmentReplicationBaseIT {
 
     @Before
     public void randomizeRemoteStoreEnabled() {
-        useRemoteStore = randomBoolean();
+        useRemoteStore = true;
```

Comment:
with https://github.com/opensearch-project/OpenSearch/pull/16760 we will limit to only remote store, but until that pr lands lets pls not hardcode this here.

Reply:
Sure